lib/tasks/coupons.rake

```diff
@@ -0,0 +1,24 @@
+namespace :coupons do
+  desc "Create coupons for all the users"
+  task create: :environment do
+    User.find_each do |user|
+      unique_code = loop do
+        random_code = Faker::Code.unique.asin
+        break random_code unless Coupon.exists?(code: random_code)
```

Comment:
create a service that takes in a class, a column name, a length and returns a unique code of that length

Reply:
can we write `break unless Coupon.exists?(code: random_code)` here?
what's the difference

Reply:
Service created.

`break unless Coupon.exists?(code: random_code)` returns nil.
So, i have to write `random_code` there to return it
break returns what we provide to it.